prisma/schema.prisma

```diff
@@ -8,6 +8,71 @@ datasource db {
   directUrl = env("DATABASE_URL_DIRECT")
 }
 
+model Event {
+  id        String @id
+  createdBy String //User id (from clerk)
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  name        String   @unique
+  location    String
+  description String
+  volunteers  String[]
+
+  startDate DateTime
+  endDate   DateTime
+
+  repairRequests RepairRequest[] //relation field, not a column (one event to many repairRequests)
+}
+
+enum Status {
+  PENDING
+  ACCEPTED
+  REJECTED
+  COMPLETED
+  CANCELLED
+}
+
+model RepairRequest {
```

Comment:
I remember talking about a request having comments, is that still the case or out of scope?

Reply:
I reckon the description will suffice for comments?